src/python/pants/backend/awslambda/python/target_types.py

```diff
@@ -117,8 +118,8 @@ async def resolve_python_aws_handler(
     return ResolvedPythonAwsHandler(f"{normalized_path}:{func}", file_name_used=True)
 
 
-class PythonAwsLambdaDependencies(Dependencies):
-    supports_transitive_excludes = True
+class PythonAwsLambdaDependencies(BinaryTargetDependencies):
+    ...
```

Comment:
Nit, we use `pass` normally for this in this repo. `...` is usually for type hints.